scripts/run.sh

```diff
@@ -12,29 +12,31 @@ then
     export WIDTH=32
     export HEIGHT=16
 else
-    export WIDTH=7
-    export HEIGHT=6
+    export WIDTH=12
```

Comment:
Any reason why this has to be 12x12? This size determines how big of the array to run for a PR. Usually you want to be the minimum size that can run all the apps required for the PR.

Reply:
I don't think so. Just changed it back to 7 x 6.